server/api_test.go

```diff
@@ -267,6 +267,18 @@ func TestHandleEndPoll(t *testing.T) {
 	expectedPost1 := &model.Post{}
 	model.ParseSlackAttachment(expectedPost1, expectedattachments1)
 
+	api9 := &plugintest.API{}
```

Comment:
My policy for naming and sorting sub tests and apis:
- Keep the tests and apis in the same orders as they are used in the code
- Bump api numbers for new tests

2/5 if this is a good policy

Reply:
To be honest, I got a bit confusing for mixed numbering of api veriables.
And since api numbers is mixed, when adding apis, next number will not be immediately understood.

But I don't have nicer idea, so I think that your policy is better realistic solution for now. 👍